service.subtitles.legendasdivx/resources/language/Portuguese (Brazil)/strings.po

```diff
@@ -0,0 +1,121 @@
+# Addon Name: LegendasDivx.com
```

Comment:
move this file to the new language folder layout:

resources/language/resource.language.pt_br/strings.po

Reply:
ok